### ISSUE TITLE
QR-Code Different Server

### ISSUE THREAD

**Issue by dev-clavis** [dev-clavis/FlatteredDoc#1]
Allow to use on different servers.
Server will be on the QR-Code.

**Comment by Craftplacer** [comment]
How would we design the QR-Code?
I would opt for an URL one with like the following syntax
`http://$SERVER/survey/$ID`

This approach would also benefit from us being able to serve a web version or lead the user to a download page if they scan it like usual.

**Comment by dev-clavis** [comment]
Yes that it will be fine, but let's assume, that we don't use advanced routing, so let's say
__http://$SERVER/survey?id=$ID__
